cloudbuild/Dockerfile

```diff
@@ -1,3 +1,19 @@
 # This Dockerfile creates an image for running presubmit tests.
 FROM openjdk:8
 
+RUN \
+  echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | \
+  tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+  curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | \
+  apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && \
+  apt-get update -y && \
+  apt-get install google-cloud-cli -y \
+  && apt clean
+# Install Python and Basic Python Tools (Assuming VM does not have them)
+RUN apt-get -y install python3 && apt clean
```

Comment:
Let's remove the `assuming` postscript. If VM doesn't have them, then it's fine to install like this. But if the VM does, then we must remove this superfluous installation.